BvgFetcher.js

```diff
@@ -1,15 +1,15 @@
 const createClient = require("hafas-client");
 const shortenStationName = require("vbb-short-station-name");
-const bvgProfile = require("hafas-client/p/bvg");
+const profile = require("hafas-client/p/bvg");
```

Comment:
i'd still call it bvgProfile since it is one :-)

Reply:
To better compare the code to the MMM-PublicTransportHafas fork, I'd like a more generic name.
Would ‘hafasProfil‘ be an acceptable compromise?

Reply:
Not needed, I understand and will merge